test/fuzz/transformation_composite_construct_test.cpp

```diff
@@ -142,10 +142,15 @@ TEST(TransformationCompositeConstructTest, ConstructArrays) {
   TransformationCompositeConstruct make_vec2_array_length_3_bad(
       37, {41, 45, 27, 27}, MakeInstructionDescriptor(46, SpvOpAccessChain, 0),
       200);
+  // This test covers line 164.
+  TransformationCompositeConstruct make_vec2_array_length_3_cover(
```

Comment:
Don't use "cover" in the name. Coverage is what's driving us to write the extra assertions, but they should be assertions that check something specific (in order to cover the new code), so the name of the transformation should reflect that; it shouldn't read like "this is just here for the sake of coverage".